scripts/uninstall_istio_system.sh

```diff
@@ -1,12 +1,10 @@
 #!/bin/bash
 echo "setting up kubeconfig"
 ./usr/local/app/scripts/init_kubeconfig.sh
-if test -f "/app/overlay-config.yaml"
-then
-    echo "Uninstalling istio and overlay file"
-    istioctl manifest generate -f /app/istio-base.yaml -f /app/overlay-config.yaml | kubectl delete -f -
+if test -f "/app/overlay-config.yaml"; then
+    echo "uninstalling istio and overlay file"
+    istioctl manifest generate -i $ISTIO_NAMESPACE -f /app/istio-base.yaml -f /app/overlay-config.yaml | kubectl delete -f - || true
```

Comment:
|| true is what keeps the script from failing if it returns a non zero exit code due to this issue https://github.com/istio/istio/issues/26234

Reply:
add flag `--ignore-not-found=true` instead to result in `0` exit code